### ISSUE TITLE
Add support for sharing lists

### ISSUE THREAD

**Issue by banool** [banool/auslan_dictionary#136]
Some kind of list export / import feature for sharing lists with friends would be great.

Doing a one time import would be easy (just export the list as JSON or something). If we wanted to support lists that update as they're changed though, that'd require some syncing mechanism. Ideally you'd be able to choose either anyone can change the list or just one person is the list updater and the rest are subscribers.

**Comment by AlliBeltane** [comment]
I would love this feature. I study Auslan in a class, and we'd absolutely share lists between us if we could

**Comment by banool** [comment]
I agree this would be a great feature. Unfortunately the ideal way to implement it probably implies having a server act as the authoritative source of info on the lists. Right now the app has no server, so I'd have to build out and pay for that infrastructure from scratch. If I get some free time I'll take a crack at it.
